### PR TITLE
add restaurant

### PR DESCRIPTION
**Pull request by mkhajure7000**

### CODE REVIEW

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -22,4 +22,4 @@ def destroy
     redirect_to root_path, notice: "Log out"
   end
 
-end
```
Comment (Shubham-Porwall) [inline app/controllers/sessions_controller.rb]: Why you do changes in this file

File: app/javascript/packs/application.js [hunk 2]
```diff
@@ -7,8 +7,6 @@ require("@rails/ujs").start()
 require("turbolinks").start()
 require("@rails/activestorage").start()
 require("channels")
-
```
Comment (Shubham-Porwall) [inline app/javascript/packs/application.js:10]: Why you do changes on these file ?

File: test/fixtures/restaurants.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
+  name: MyString
+  address: MyString
```
Comment (Shubham-Porwall) [inline test/fixtures/restaurants.yml]: Again you are pushing unnecessary files. Why you are doing same mistake again and again.
cc: @kalpana19sharma

File: app/controllers/sessions_controller.rb (hunk 1 above)
Comment (mkhajure7000) [reply app/controllers/sessions_controller.rb]: I had to show messages using flash , so i used flash instead of notice

Comment (Shubham-Porwall) [reply app/controllers/sessions_controller.rb]: See the files changes once, Its showing you only add one space.

File: test/models/restaurant_test.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class RestaurantTest < ActiveSupport::TestCase
+  # test "the truth" do
```
Comment (Shubham-Porwall) [inline test/models/restaurant_test.rb]: If I said remove unnecessary files you have to remove all unnecessary files.
Why you always remove only those file which i commented on? 

I think It's 5th time i am explaining you !
image

**Review by Shubham-Porwall — APPROVED** [review]

**Pull request merged**